packages/grid/x-data-grid-generator/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-data-grid-generator",
-  "version": "5.17.15",
+  "version": "5.17.16",
```

We should fix our mistake and release this package in sync with other data grid packages.
```suggestion
  "version": "5.17.17",
```